dev-docs/bidders/exadsadserver.md

```diff
@@ -0,0 +1,487 @@
+---
+layout: bidder
+title: EXADS
+description: EXADS Bidder Adapter
+pbjs: true
+pbs: false ?
+biddercode: exadsadserver
+gvl_id: 1084 ? // found here https://iabeurope.eu/vendor-list-tcf/
+tcfeu_supported: true?
+usp_supported: false?
+media_types: banner, video, native
+gpp_sids: ?
+schain_supported: ?
+dchain_supported: ?
+safeframes_ok: ?
+deals_supported: ?
+floors_supported: ?
+ortb_blocking_supported: ?
```

Comment:
@broucz [The link is that mentioned above:  https://docs.prebid.org/dev-docs/bidder-adaptor.html.](https://github.com/EXADS/prebid.github.io/pull/2#issue-2175894774)

Reply:
I would remove those where we know that default matches what we have, and for the other we should put correct values:

```
pbs: false,
gvl_id: 1084, but confirm with Barry/Tiago if everyone uses EXADS vendor id or they apply for their own,
tcfeu_supported: true,
usp_supported: false,
gpp_sids: tcfeu,
schain_supported: false,
dchain_supported: false,
safeframes_ok: false (have no idea tbh, but since we never tried I would go with false),
deals_supported: false,
floors_supported: true,
ortb_blocking_supported: true - not yet live but I'm pretty sure it will go live before Prebid team gets hands on this
```

Reply:
Changed all except glv_id for which I'm waiting feedback by @tfoliveira and Barry about gvl_id into slack.

Reply:
@tfoliveira has confirmed that glv_id is 1084.